provisioning/configuration/roles/buildbot-master/tasks/main.yml

```diff
@@ -82,6 +82,8 @@
       value: '{{wptd_legacy_notify_secret}}'
     - name: wptd_upload_url
       value: '{{wptd_upload_url}}'
+    - name: wptd_staging_upload_url
+      value: '{{wptd_staging_upload_url}}'
```

Comment:
@Hexcles Buildbot has a notion of "secret" values which it will redact from publicly-visible logging output. It can read these values from flat files or from a Vault server. I chose the former to avoid the complexity that Vault would introduce, but it means that we need to maintain this unfortunate bit of duplication.